util/log/log.go

```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const timeFormat = "1/2 15:04:05.000" // m/d h:m:s.nanos
```

Comment:
we should really have YYYY/MM/DD as the date format.
Actually, time.RFC3339 seemed reasonable.